applications/nrf5340_audio/src/modules/sd_card.c

```diff
@@ -47,7 +47,11 @@ int sd_card_list_files(char const *const path, char *buf, size_t *buf_size)
 		return -EPERM;
 	}
 
-	k_sem_take(&m_sem_sd_oper_ongoing, K_MSEC(K_SEM_OPER_TIMEOUT_MS));
+	ret = k_sem_take(&m_sem_sd_oper_ongoing, K_MSEC(K_SEM_OPER_TIMEOUT_MS));
+
+	if (ret != 0) {
+		LOG_ERR("Sem take failed. Ret: %d", ret);
```

Comment:
Is this debug code? If not - should it return and not execute the rest of the function?